web/src/beta/features/PluginPlayground/Console/index.tsx

```
@@ -1,7 +1,34 @@
+        title: string;
+        output: string;
+      }[]
+    | undefined;
 };
 
+const Console: FC<Props> = ({ fileOutputs }) => {
+  if (!fileOutputs) return null;
+  return (
+    <Wrapper>
+      {fileOutputs.map(({ title, output }) => (
+        <div>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)